### DISCUSSION TITLE
Stellarium won’t consult a specific survey

### DISCUSSION THREAD

**Discussion by RandyF904** [Stellarium/stellarium#4004] [General]
In Stellarium I want to include a star listed in survey 2mass6x.  How do I do that?    User guide indicates that in sky view that survey would show.  No joy.  And typing the coordinates into the search tool, nothing.

**Comment by gzotti** [comment]
The HiPS surveys in Stellarium are only raster images, no object catalogs. You cannot identify stars by number or such. 
To see the HiPS rendered, you need to select it in the HiPS dialog (User Guide 4.4.7) and then activate the actual HiPS drawing. You may need to enable the corresponding button first (User Guide 4.3.3).

**Reply by RandyF904** [reply]
Thanks!

**Comment by RandyF904** [comment]
I am a little perplexed. I am doing research to help someone considering a
potential exoplanet. More information is here.
https://exofop.ipac.caltech.edu/tess/target.php?id=180672438  It is a
variable star. My plan was to identify it in Stellarium and slave my
telescope to it, completing my research.  More information is here.


Is there anyway Stellarium can help me?

**Reply by gzotti** [reply]
Exoplanets are listed and searchable in the Exoplanets plugin.
